apps/src/code-studio/components/ModelCard.jsx

```diff
@@ -113,7 +113,8 @@ const styles = {
     borderRadius: 5,
     borderColor: color.gray,
     marginBottom: 10,
-    padding: 10
+    padding: 10,
+    overflowWrap: 'breakWord'
```

Comment:
Did this work?  I think it should be `"break-word"`.  (You can double-check it's valid by inspecting the element in the browser debugger.)

Reply:
Ah, good catch! I conflated this with the spacing change. Now it's showing up in the browser debugger. Thank you!